src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -62,11 +76,44 @@ def __post_init__(self):
                     f"Failed to load {self.name}: {err}"
                 ) from err
 
+    @staticmethod
+    def _make_arg_types_list_new(content_dict: ConfigDict) -> List[ContentTypeEnum]:
+        # First find the number of args
+        args_spec: List[Tuple[int, str]] = content_dict.get(
+            WorkflowJobKeys.ARG_TYPE, []
+        )  # type: ignore
+        specified_highest_arg_index = (
+            max(index for index, _ in args_spec) if len(args_spec) > 0 else -1
+        )
+        specified_max_args: int = content_dict.get("MAX_ARG", 0)  # type: ignore
+        specified_min_args: int = content_dict.get("MIN_ARG", 0)  # type: ignore
+
+        num_args = max(
+            specified_highest_arg_index + 1,
+            specified_max_args,
+            specified_min_args,
+        )
+
+        arg_types_dict: Dict[int, ContentTypeEnum] = dict()
+
+        for i, type_as_string in args_spec:
+            # make old tests pass temporarily, will use
```

Comment:
I think we should avoid this "future todo" comments. They get outdated fast.

Reply:
removed